db_s3_backup.py

```diff
@@ -218,7 +174,7 @@ def delete_local_backups(dir_path, backup_prefix, backup_extension, verbose = Fa
 		default=False,
 	)
 	args = parser.parse_args()
-	
+
 	# Loads configuration file from JSON
 	try:
 		f=open(args.config_file, 'r')
```

Comment:
ideally one day we should turn all these old style `open()` to use context managers `with…:`